client/http-client/Cargo.toml

```diff
@@ -29,3 +29,10 @@ tokio = { version = "1.16", features = ["net", "rt-multi-thread", "macros"] }
 [features]
 default = ["tls"]
 tls = ["hyper-rustls/webpki-tokio"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
What does this do?

Reply:
By default `docs.rs` only builds the docs with default-features so we want all features to be documented.

Reply:
Ah thanks, and the `--cfg docsrs` bit is about making it possible to conditionally compile stuff for doc purposes I gather!

Reply:
should be equivalent to `RUSTDOCFLAGS="--cfg docrs" cargo doc --workspace --no-deps --document-private-items` it seems.